index.js

```diff
@@ -1,11 +1,17 @@
+
+Yeah im already feeling super discouraged so sorry for submitting this late 
+
 // ***** Question 1 *****
+var counter = 1
+
+ var counter = 2
```

Comment:
In general we want to avoid using `var` because it has some tricky behavior. For this we'd want to use `let` since it allows us to reassign the variable to a new value:

```js
let counter = 1
counter = 2
```